src/commands/quote.ts

```diff
@@ -0,0 +1,105 @@
+import { CommandInteraction } from 'discord.js'
+import { Discord, SimpleCommand, SimpleCommandMessage, SimpleCommandOption, Slash, SlashOption } from 'discordx'
+import fetch from 'node-fetch'
+
+interface Quote {
+  id: number
+  body: string
+}
+
+interface QuoteData {
+  error: boolean
+  type: number
+  total: number
+  data: Array<Quote>
+}
+
+@Discord()
+class quoteCommand {
+  private endpoint = 'https://api.bufutda.com/bot/quote?channel=bananasaurus_rex'
+  private quotes: Array<Quote> = []
+  private lastFetch: Date = new Date()
+  private cacheDuration = 1 * 60 * 1000 // 30 minutes in milliseconds
```

Comment:
Make sure to have this be 30 minutes and not 1

Reply:
Oops, I still had this set to one from testing. Fixed it now.